python_modules/libraries/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing Libraries
+Welcome! We appreciate your interest in contributing to Dagster Integrations.
+This guide outlines the steps for community members who wish to contribute to
+integratiosn to the project.
+
+## Dagster Libraries
+Dagster Libraries are integrations that can be installed separately from the
+main Dagster package, and provide additional functionality with external tools,
+services, and APIs.
+
+Dagster Libraries may be maintained by either the **Dagster Team** or by
+**Dagster Community** members. Dagster libriaries will include a
+`integration.yaml` in each subfolder, which define, among other things,
+the maintainer of each package.
+
+## integration.yaml
+The `integration.yaml` file must include the following:
+
+```
+package-name: The name of the python package, for example, `dagster-duckdb`
+name: The name of the library, which may be different than the package name
+summary: Additional details about this library and what it does
```

Comment:
We might also want the source of truth on this to be the setup.py so that it will show up in PyPI.